### PR TITLE
ZH-266 - Addition of ORR document

### PR DESCRIPTION
**Pull request by M-casado**
## Summary
Added an updated version of the Organisational Roles and Responsibilities (ORR) document and updated README diagram

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change which adds new content)
- [x] Modified content (non-breaking change which modifies existing content)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Motivation and Context
ORR needs to be part of the repo

### References
[ZenHub #266](https://app.zenhub.com/workspaces/t43-gdi-sops-662a0b0fe38b0405fb600d67/issues/zh/266)

## Changes Introduced
- Added an updated version of the ORR doc at ``docs/GDI-SOP_organisational-roles-and-responsibilities.md``
- Updated the main diagram in the ``README.md``

## Review
The ORR document was reviewed before, but I updated it a bit more.

## Additional Notes
NA

## Checklist:

**General Compliance:**
- [x] My changes follow the code style of this project ([GDI SOP Style Guide](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_style-guide.md)) and the file naming conventions of the [file accessioning proposal](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs//GDI-SOP_sop-accessioning.md).
- [x] I have verified that all new updated content is accessible, including checking that all external references are readable (i.e., no broken links). These may include references to external resources that should be resolvable, and internal references among SOPs.
- [ ] I have properly added this PR's changes to the repository [CHANGELOG.md](../CHANGELOG.md).

Only applicable if the PR includes new, or changes to, GDI SOPs (i.e., documents at ``sops/``):
- [ ] My SOP-related changes adhere to the [Generic SOP Template](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_sop-template.md), including format and required fields.
- [ ] I have consulted the [Charter](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_charter.md), [ISM](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_information-service-management.md), and [ORR](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_organisational-roles-and-responsibilities.md) documents to ensure compliance. 
- [ ] I am complying with the established procedure for SOP creations and modifications, including respecting review phases and notifying needed contributors for reviews.

### CODE REVIEW

File: docs/GDI-SOP_organisational-roles-and-responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+| **Reviewer** | European and Node level | - **Review SOP Templates** for accuracy and compliance. <br> - Suggest improvements and ensure alignment with GDI objectives. <br> - Facilitate inter-node communication. | - Broad understanding of GDI's objectives. <br> - Detail-oriented with a critical eye for issues. <br> - Experience in collaborative reviews and multi-disciplinary teams. |
+| **Approver** | European and Node level | - **Confirm SOP Templates readiness** post-review. <br> - **Ensure compliance** with **GDI standards and GDPR**. <br> - Document and store SOP Templates in designated platforms (i.e., GitHub). <br> - Oversee SOP lifecycle and ensure periodic reviews. <br> - Transfer approved SOPs to GitHub for version control.  | - Leadership and decision-making capabilities. <br> - Deep understanding of GDI strategic goals. <br> - Proficiency in managing project documentation platforms. <br> - Experience in document lifecycle management. |
+| **Authorizer** | European level | - **Grant (or veto) official permission for SOPs to be released**. | - High-level management skills. <br> - Understanding of GDI scope and operations.  |
```
Comment (GabiRinck) [inline docs/GDI-SOP_organisational-roles-and-responsibilities.md:45]:
```suggestion
| **Authorizer** | European level | - **Grant (or veto) official permission for SOPs to be released**. | - High-level management skills. <br> - Part of the GDI executive leadership. <br> - Understanding of GDI scope and operations. <br> - Time-management skills: where necessary, exercise veto rights within a set time period.  |
```

File: docs/GDI-SOP_organisational-roles-and-responsibilities.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| **Main repository Maintainer (OC/SDPC)** | European level | - **Oversee SOP creation and review process** for the GDI GitHub repository. <br> - **Coordinate with Authors, Reviewers, Approvers and Authorizers** to ensure SOP templates meet GDI standards. <br> - Manage version control, ensure updates are made consistently. <br> - **Conduct periodic audits** of the SOP repository to maintain quality and compliance. | - Strong project management and oversight skills. <br> - Proficiency in GitHub repository management, including version control and collaborative document handling. <br> - Comprehensive understanding of GDI operations and objectives. <br> - Ability to work effectively with multidisciplinary teams and manage stakeholder expectations. |
```
Comment (GabiRinck) [inline docs/GDI-SOP_organisational-roles-and-responsibilities.md:46]: Isn't there a certain overlap with the responsibilities of the Approver (version control & overseeing SOP life-cycle)??

File: docs/GDI-SOP_organisational-roles-and-responsibilities.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+| **Trainer** | European and Node level | - **Conduct training** on SOP Templates for relevant stakeholders. <br> - Ensure understanding of procedures and compliance requirements. <br> - Provide updates on SOP revisions. | - In-depth knowledge of SOPs and the GDI context. <br> - Strong communication and training skills. <br> - Experience in adult education and training methodologies.        |
+| **Node SOP Maintainer** | Node level | - **Monitor for updates** to GDI SOP Templates. <br> - **Ensure node-specific SOPs align with updates**. <br> - Gather feedback on SOP implementation and relay to development team. | - Proficient in GitHub version control. <br> - Detail-oriented with the ability to assess the impact of updates. <br> - Strong communication skills for feedback gathering. |
```
Comment (GabiRinck) [inline docs/GDI-SOP_organisational-roles-and-responsibilities.md:48]:
```suggestion
| **Node SOP Maintainer** | Node level | - **Monitor for updates** to GDI SOP Templates. <br> - **Ensure node-specific SOPs align with updates**. <br> - Gather feedback on SOP implementation, any potential issues and relay to the SOP development team. | - Proficient in GitHub version control. <br> - Detail-oriented with the ability to assess the impact of updates. <br> - Strong communication skills for feedback gathering. |
```

**Review by jmenglund — APPROVED** [review]

File: docs/GDI-SOP_organisational-roles-and-responsibilities.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# European GDI - Organisational Roles and Responsibilities
+
+## 1. Document History
+| Version | Author(s) | Description of changes | Date |
+| -- | -- | -- | -- |
+| v1 | Marcos Casado Barbero | First ORR document released in the GH repository | 2024.09.25 |
+| v0 | Mallory Freeberg, Marcos Casado Barbero | First draft of the ORR document | 2024.03.12 |
+
+## 2. Glossary
+Find GDI SOPs common Glossary at the [charter document](./GDI-SOP_charter.md#2-glossary).
```
[the remark this answers is not in the document]
Comment (M-casado) [reply docs/GDI-SOP_organisational-roles-and-responsibilities.md:10]: Will do 👍

File: docs/GDI-SOP_organisational-roles-and-responsibilities.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+| **Reviewer** | European and Node level | - **Review SOP Templates** for accuracy and compliance. <br> - Suggest improvements and ensure alignment with GDI objectives. <br> - Facilitate inter-node communication. | - Broad understanding of GDI's objectives. <br> - Detail-oriented with a critical eye for issues. <br> - Experience in collaborative reviews and multi-disciplinary teams. |
+| **Approver** | European and Node level | - **Confirm SOP Templates readiness** post-review. <br> - **Ensure compliance** with **GDI standards and GDPR**. <br> - Document and store SOP Templates in designated platforms (i.e., GitHub). <br> - Oversee SOP lifecycle and ensure periodic reviews. <br> - Transfer approved SOPs to GitHub for version control.  | - Leadership and decision-making capabilities. <br> - Deep understanding of GDI strategic goals. <br> - Proficiency in managing project documentation platforms. <br> - Experience in document lifecycle management. |
```
[the remark this answers is not in the document]
Comment (M-casado) [reply docs/GDI-SOP_organisational-roles-and-responsibilities.md:44]: Done

File: docs/GDI-SOP_organisational-roles-and-responsibilities.md (hunk 2 above)
Comment (M-casado) [reply docs/GDI-SOP_organisational-roles-and-responsibilities.md:46]: Addressed above

**Pull request merged**